### ISSUE TITLE
Please don't insert url_key attribute values if the url_key on a specific storeview isn't the same as in the default values

### ISSUE THREAD

**Issue by hostep** [olegkoval/magento2-regenerate_url_rewrites#92]
Hi!

This was tested on a Magento shop with 2 store views.
When you have a product with a url_key set to `something` on the default storeview but not overwriting it on a specific storeview scope, then after running this tool, it inserts the url_key value `something` on the 2 storeviews.

This makes it hard to update the url_key for a product, since you then need to manually go through all store views, and update all url_key's.

So if a url_key isn't filled in on a specific store view, it should just inherit it from the default values, and not save it separately in the database.

Thanks!

**Comment by werfu** [comment]
As explained in #pr78, this is default behavior. My PR was refused for this reason. However, I will submit soon a PR with the reverse option, that is not to regen URL_key.

**Comment by hostep** [comment]
Thanks for replying @werfu, I think you are talking about #93.

This ticket (92) is about not generating url keys on storeview level if it already exists on default level.
This is very related, but not exactly the same :)

**Comment by werfu** [comment]
Yeah, I mixed up. I replied in the other thread for clarity purpose.

About this one though, I'm not sure how, but we should check if the current storeview key is using the default or not. In some case, the generated url_key will not be the same as the attribute used for generating the key will be overridden.

**Comment by olegkoval** [comment]
Implemented in 1.8.0: when regenerating url_key (--regen-url-key), a redundant per-store override is no longer written if the regenerated value is identical to the default-scope value - avoids needlessly bloating the EAV varchar tables with duplicate per-store rows.
